### PR TITLE
ac: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/ac.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+`ac`
+
+- Imprime cuánto tiempo han estado conectados los usuarios en horas:
+
+`ac -p`
+
+- Imprime cuánto tiempo ha estado conectado un usuario en particular en horas:
+
+`ac -p {{nombreusuario}}`
```
Comment (kbdharun) [inline pages.es/common/ac.md:16]:
```suggestion
`ac -p {{nombre_de_usuario}}`
```

File: pages.es/common/ac.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Imprime cuánto tiempo ha estado conectado un usuario en particular en horas por día (con total):
+
+`ac -dp {{nombreusuario}}`
```
Comment (kbdharun) [inline pages.es/common/ac.md:20]:
```suggestion
`ac -dp {{nombre_de_usuario}}`
```

**Review by kbdharun — COMMENTED** [review]
Generally, `nombre_de_usuario` is what we use for `username` in Spanish pages. [Ref](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

File: pages.es/common/ac.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# ac
+
+> Imprimir estadísticas sobre cuánto tiempo han estado conectados los usuarios.
```
Comment (navarroaxel) [inline pages.es/common/ac.md:3]: Based on
> Cuánto se escribe con tilde cuando tiene función interrogativa o exclamativa. Mientras que cuanto, cuando se emplea como adjetivo, adverbio o pronombre, no lleva acento gráfico.
```suggestion
> Imprimir estadísticas sobre cuanto tiempo han estado conectados los usuarios.
```

File: pages.es/common/ac.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+> Más información: <https://man.openbsd.org/ac>.
+
+- Imprime cuánto tiempo ha estado conectado el usuario actual en horas:
```
Comment (navarroaxel) [inline pages.es/common/ac.md:6]:
```suggestion
- Imprime cuanto tiempo ha estado conectado el usuario actual en horas:
```

File: pages.es/common/ac.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`ac`
+
+- Imprime cuánto tiempo han estado conectados los usuarios en horas:
```
Comment (navarroaxel) [inline pages.es/common/ac.md:10]:
```suggestion
- Imprime cuanto tiempo han estado conectados los usuarios en horas:
```

File: pages.es/common/ac.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`ac -p`
+
+- Imprime cuánto tiempo ha estado conectado un usuario en particular en horas:
```
Comment (navarroaxel) [inline pages.es/common/ac.md:14]:
```suggestion
- Imprime cuanto tiempo ha estado conectado un usuario en particular en horas:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**